client/components/nts.js

```diff
@@ -1,19 +1,28 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+import { useSelector } from 'react-redux';
 import { Button } from '@ukhomeoffice/react-components';
 
-const NTS = ({ accepted, onAccept, review, hideTitle }) => (
-  <div className="nts">
-    {
-      !hideTitle && <h3>Creating your non-technical summary</h3>
-    }
-    <p>Any information you enter on this screen will form part of a non-technical summary (NTS) on GOV.UK, so use everyday language that can be easily understood by members of the public.</p>
-    <p>As an NTS is a public document, do not include any information that could identify any people, locations, or intellectual property related to your project. You will be able to review and edit your NTS as a whole before you send your licence application to the Home Office.</p>
-    {
-      !review && (
-        <Button disabled={accepted} onClick={onAccept}>Continue</Button>
-      )
-    }
-  </div>
-);
+export default function NTS({ accepted, onAccept, review, hideTitle }) {
+  const isFullApplication = useSelector(state => state.application.isFullApplication);
 
-export default NTS;
+  return (
+    <div className="nts">
+      {
+        !hideTitle && <h3>Creating your non-technical summary</h3>
+      }
+      {
+        !isFullApplication && (
```

Comment:
Is there a context we want to render _anything_ in this component is `isFullApplication` is true?

If not then I'd prefer to return an early `null` rather than relying on the other props being set correctly. It feels a little fragile.

Reply:
yeah I think we can return early here